src/constant/enums.js

```diff
@@ -81,6 +81,40 @@ export function getParkingRequestStatus(parkingRequestStatus) {
     return getEnumLabel(ParkingRequestStatusLabel, parkingRequestStatus);
 }
 
+export const SpotRequestStatus = Object.freeze({
```

Comment:
This enum values are not random.
It should be in sync with backend enum values
https://github.com/parkspot-dev/maya/blob/master/core/models/struct.go#L562